lib/reek/ast/sexp_extensions/send.rb

```diff
@@ -10,6 +10,10 @@ def receiver
           children.first
         end
 
+        def value_assigned
```

Comment:
Not a fan of this name I have to say. Shouldn't it be "assigned_value"? Also, we're talking about the "right hand assigment", aren't we? Then how about calling it "right_hand_assignment"?